src/Game/UI/Gumps/OptionsGump.cs

```diff
@@ -556,9 +547,88 @@ private void BuildVideo()
+                _darkNights.IsChecked = false;
+                _runMouseInSeparateThread.IsChecked = true;
+
+                switch (option)
+                {
+                    case 0: // High
+                        _sliderFPS.Value = Constants.MAX_FPS;
+                        _zoomCheckbox.IsChecked = true;
+                        _useColoredLights.IsChecked = true;
+                        _enableDeathScreen.IsChecked = true;
+                        _enableBlackWhiteEffect.IsChecked = true;
+                        _enableShadows.IsChecked = true;
+                        _auraType.SelectedIndex = 3;
+                        _auraMouse.IsChecked = true;
+                        _xBR.IsChecked = true;
+                        _hideChatGradient.IsChecked = false;
+                        break;
+                    case 1: // Medium
+                        _sliderFPS.Value = 25;
+                        _zoomCheckbox.IsChecked = false;
+                        _useColoredLights.IsChecked = true;
+                        _enableDeathScreen.IsChecked = true;
+                        _enableBlackWhiteEffect.IsChecked = true;
+                        _enableShadows.IsChecked = false;
+                        _auraType.SelectedIndex = 1;
+                        _auraMouse.IsChecked = false;
+                        _xBR.IsChecked = false;
+                        _hideChatGradient.IsChecked = false;
+                        break;
+                    case 2: // Low
+                        _sliderFPS.Value = Constants.MIN_FPS;
+                        _gameWindowWidth.SetText("800");
+                        _gameWindowHeight.SetText("600");
+                        _zoomCheckbox.IsChecked = false;
+                        _useColoredLights.IsChecked = false;
+                        _enableDeathScreen.IsChecked = false;
+                        _enableBlackWhiteEffect.IsChecked = false;
+                        _enableShadows.IsChecked = false;
+                        _auraType.SelectedIndex = 0;
+                        _auraMouse.IsChecked = false;
+                        _xBR.IsChecked = false;
+                        _hideChatGradient.IsChecked = true;
+                        break;
+                }
+            };
+
+            profileItem.Add(text);
+            profileItem.Add(perfProfile);
+            rightArea.Add(profileItem);
+
+            Add(rightArea, PAGE);
+
+            Add(new Line(190, rightArea.Height + 10, WIDTH - 210, 1, Color.Gray.PackedValue), PAGE);
+
+            rightArea = new ScrollArea(190, rightArea.Height + 20, WIDTH - 210, 340, true);
+
+            ScrollAreaItem fpsItem = new ScrollAreaItem();
+            text = new Label("- FPS:", true, HUE_FONT);
+            fpsItem.Add(text);
+            _sliderFPS = new HSliderBar(text.X + 90, 5, 250, Constants.MIN_FPS, Constants.MAX_FPS, Settings.GlobalSettings.FPS, HSliderBarStyle.MetalWidgetRecessedBar, true, FONT, HUE_FONT);
```

Comment:
Maybe `_sliderFPS` can be renamed to `_sliderFps` since there is already `fpsItem` and various IDE tooling is not fond of `FPS` unless its in some custom Dictionary (which we technically have with .DotSettings.